Coop/styles/Vocab/CoopAbbreviations/accept.txt

```diff
@@ -1,6 +1,7 @@
 AAD
-Ajax
+[aA]jax
```

Comment:
Why do we need the alternate spelling?

Reply:
I got an error with ajax in the SAP neptune guidelines.

Reply:
I took a look at the error in the build, in both of the Ajax cases I would fix the error by replacing ajax with Ajax, if you are referencing a specific directory, file or command I recommend using monospace.

You get monospace like this:

```markdown
`ajax`
```

Reply:
https://developer.mozilla.org/en-US/docs/Web/Guide/AJAX

Reply:
Ok :-)